tests/TestCase.php

```diff
@@ -2,6 +2,41 @@
 
 namespace ipl\Tests\Web;
 
+use ipl\Html\ValidHtml;
+use PHPUnit\Util\Xml;
+
+if (class_exists('PHPUnit_Util_XML')) {
+    // Support older PHPUnit versions
+    class_alias('PHPUnit_Util_XML', 'PHPUnit\\Util\\Xml');
+}
+
 abstract class TestCase extends \PHPUnit\Framework\TestCase
 {
+    /**
+     * Assert that HTML is equal
+     *
+     * @param string    $expectedHtml
+     * @param ValidHtml $actual
+     * @param String    $message
+     *
+     * @return void
+     */
+    protected function assertHtml(string $expectedHtml, ValidHtml $actual, String $message = ''): void
```

Comment:
The `string` type name always all lowercase please.

Reply:
Oh well, nevermind. Please extend `ipl\Tests\Html\TestCase` in `CompatFormTest`. Then this isn't required at all.